### ISSUE TITLE
MenuBar without AutoDisplay: menu doesn't close when clicked

### ISSUE THREAD

**Issue by NiarK** [primefaces/primeng#5196]
**I'm submitting a ...**  (check one with "x")
```
[ x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Bug Reports**
The case demonstrating this bug report can be found on the Primeng MenuBar preview:
https://www.primefaces.org/primeng/#/menubar

**Current behavior**
When AutoDisplay is set to false, the menu does not close when clicking on a sub menu item.

**Expected behavior**
When clicking on a menu item of a MenuBar with AutoDisplay is set to false, the menu must close.

**Minimal reproduction of the problem with instructions**
The case demonstrating this bug report can be found on the Primeng MenuBar preview:
https://www.primefaces.org/primeng/#/menubar.
On the "Without AutoDisplay" example :
 - Click on the menu "File" -> "Open : the menu does not close (like it does when AutoDisplay is set to true).
 - Click on the sub menu item "File" -> "New" -> "Project" : the "New" sub menu does close but not the main "File" menu.

**What is the motivation / use case for changing the behavior?**
AutoDisplay set to false cannot be use.

**Please tell us about your environment:**
Windows 7 64 bits, Firefox ESR 52.6.0 (32 bits)

* **Angular version:** 5.1.0

* **PrimeNG version:** 5.2.0

* **Browser:** At least on Firefox ESR 52.6.0 (32 bits)

**Comment by the-golden-girl** [comment]
I am also facing the same issue.

**Comment by TineyeTarball** [comment]
This seems to be fixed in 5.2.3

**Comment by jotae1e** [comment]
Same in 7.1.0
